lib/std/elf.zig

```diff
@@ -1481,6 +1481,38 @@ pub const EM = enum(u16) {
     _BPF = 247,
 
     _,
+
+    pub fn toTargetarch(em: EM) std.target.Arch {
```

Comment:
Ditto.
```suggestion
    pub fn toTargetCpuArch(em: EM) std.Target.Cpu.Arch {
```

Reply:
Cheers! Checking in fixes now! Always miss this kind of stuff with simple changes :facepalm: